### DISCUSSION TITLE
Copy-Paste Functionality Issue in Firefox on Synology NAS

### DISCUSSION THREAD

**Discussion by seriouspike** [jlesage/docker-firefox#230] [Q&A] [answered]
Hi,

I've installed jlesage/firefox:latest on my Synology. I can open the Firefox browser and have an internet connection.

The problem is that I don't have copy-paste functionality. When I try to copy text into the Firefox tab from other tabs or from outside the Firefox tab, it doesn't work. Is there an option I need to enable? What is the solution?

**Comment by jlesage** [comment]
Are you using the clipboard from the side-panel ?

**Reply by seriouspike** [reply]
Hi,
Yes, from the side panel it works, but why I cant use normal clipboard?

**Reply by jlesage** [reply]
This is a browser limitation: for security reason, most browsers don't allow web page code to seamlessly access the clipboard, without some kind of user intervention.

**Reply by seriouspike** [reply]
How I can change this in Chrome?

**Reply by jlesage** [reply, marked as answer]
You mean how to change the browser limitation ?  You can't...

Chrome can ask user to allow permission to access the clipboard, but this mechanism needs to be supported by the web page (which is not the case for this container).  The my knowledge, all other browsers don't support this.

**Comment by seoetraffic** [comment]
Hi,
I have the same question. How can I copy and paste from my local computer to firefox on container and copy paste from firefox on container to my local computer?

thanks!
